README.md

```diff
@@ -1,2 +1,31 @@
-# template-validator
-A tool used to validate a Job Template
+# Template Validator
+[![Version][npm-image]][npm-url] ![Downloads][downloads-image] [![Build Status][status-image]][status-url] [![Open Issues][issues-image]][issues-url] [![Dependency Status][daviddm-image]][daviddm-url] ![License][license-image]
+
+> A module for validating a Screwdriver Template file
+
+## Usage
+
+```bash
+npm install screwdriver-template-validator
+```
+
+## Testing
+
+```bash
+npm test
+```
+
+## License
+
+Code licensed under the BSD 3-Clause license. See LICENSE file for terms.
+
+[npm-image]: https://img.shields.io/npm/v/screwdriver-template-validator.svg
+[npm-url]: https://npmjs.org/package/screwdriver-template-validator
+[downloads-image]: https://img.shields.io/npm/dt/screwdriver-template-validator.svg
+[license-image]: https://img.shields.io/npm/l/screwdriver-template-validator.svg
+[issues-image]: https://img.shields.io/github/issues/screwdriver-cd/template-validator.svg
+[issues-url]: https://github.com/screwdriver-cd/template-validator/issues
+[status-image]: https://cd.screwdriver.cd/pipelines/pipelineid/badge
```

Comment:
If you have a pipeline for this repo, you can fix the link here by replacing the `pipelineid`